exporters/elasticsearch/CMakeLists.txt

```diff
@@ -1,31 +1,36 @@
-include_directories(include)
-include_directories(${CMAKE_SOURCE_DIR}/ext/include)
-
 add_library(opentelemetry_exporter_elasticsearch_logs src/es_log_exporter.cc)
 
+set_target_properties(opentelemetry_exporter_elasticsearch_logs
+                      PROPERTIES EXPORT_NAME elasticsearch_log_exporter)
+
+target_include_directories(
+  opentelemetry_exporter_elasticsearch_logs
+  PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>"
+         "$<INSTALL_INTERFACE:include>")
```

Comment:
Should this line to `${CMAKE_SOURCE_DIR}/ext/include` above?

Reply:
`opentelemetry_exporter_elasticsearch_logs` has already link `http_client_curl` , which will add `${CMAKE_SOURCE_DIR}/ext/include` into include directories.